actors/user/src/test/java/org/sunbird/user/UserRoleActorTest.java

```diff
@@ -0,0 +1,180 @@
+
+import akka.actor.ActorRef;
+import akka.actor.ActorSelection;
+import akka.actor.ActorSystem;
+import akka.actor.Props;
+import akka.testkit.javadsl.TestKit;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletionStage;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.sunbird.actor.router.RequestRouter;
+import org.sunbird.actor.service.BaseMWService;
+import org.sunbird.actorutil.InterServiceCommunication;
+import org.sunbird.actorutil.InterServiceCommunicationFactory;
+import org.sunbird.actorutil.org.impl.OrganisationClientImpl;
+import org.sunbird.cassandraimpl.CassandraOperationImpl;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.ActorOperations;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.request.Request;
+import org.sunbird.common.responsecode.ResponseCode;
+import org.sunbird.helper.ServiceFactory;
+import org.sunbird.learner.actors.role.dao.impl.RoleDaoImpl;
+import org.sunbird.models.organisation.Organisation;
+import org.sunbird.user.actors.UserRoleActor;
+import scala.concurrent.duration.Duration;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({
+  ServiceFactory.class,
+  RoleDaoImpl.class,
+  BaseMWService.class,
+  ActorSelection.class,
+  ActorRef.class,
+  RequestRouter.class,
+  InterServiceCommunication.class,
+  Organisation.class,
+  OrganisationClientImpl.class,
+  InterServiceCommunicationFactory.class,
+  Response.class
+})
+@PowerMockIgnore({"javax.management.*"})
+public class UserRoleActorTest {
+
+  private static ActorSystem system;
+  private static final Props props = Props.create(UserRoleActor.class);
+  private static CassandraOperationImpl cassandraOperation;
+  private static RoleDaoImpl roleDao;
+  private static ActorSelection actorSelection;
+  private static CompletionStage completionStage;
+  private static ActorRef actorRef;
+  private static InterServiceCommunication interServiceCommunication;
+  private static Organisation organisation;
+  private static Response response;
+  private static Map<String, Object> map;
+
+  @Before
+  public void beforeEachTest() {
+
+    PowerMockito.mockStatic(ServiceFactory.class);
+    cassandraOperation = mock(CassandraOperationImpl.class);
+    when(ServiceFactory.getInstance()).thenReturn(cassandraOperation);
+    PowerMockito.mockStatic(RoleDaoImpl.class);
+    roleDao = Mockito.mock(RoleDaoImpl.class);
+    when(RoleDaoImpl.getInstance()).thenReturn(roleDao);
+    system = ActorSystem.create("system");
```

Comment:
not required to initialize before each test, this can be initialized only once at top